java/src/main/java/com/scalar/admin/AdminCommand.java

```diff
@@ -59,7 +68,31 @@ public static void main(String[] args) {
 
   @Override
   public Integer call() {
-    RequestCoordinator coordinator = new RequestCoordinator(srvServiceUrl);
+    RequestCoordinator coordinator;
+
+    if (srvServiceUrl != null) {
```

Comment:
```suggestion
    if (srvServiceUrl != null && addresses != null) {
      throw new IllegalArgumentException(
          "Specify only either [--srv-service-url, -s] or [--addresses, -a].");
    } else if (srvServiceUrl != null) {
```
I think `--srv-service-url` and `--addresses` conflict.
So, I think it is better to check if both flags are specified and return an error if both are specified.
What do you think?

Reply:
@kota2and3kan 

It's a good idea.
I think this can prevent ambiguity.